docs/reference-guides/theme-json-reference/theme-json-living.md

```diff
@@ -172,6 +172,20 @@ Settings related to typography.
 Generate custom CSS custom properties of the form `--wp--custom--{key}--{nested-key}: {value};`. `camelCased` keys are transformed to `kebab-case` as to follow the CSS property naming schema. Keys at different depth levels are separated by `--`, so keys should not include `--` in the name.
 
 
+---
+
+### enabled
+
+Defines whether the lightbox is enabled or not.
+
+
+---
+
+### allowEditing
+
+Defines whether to show the Lightbox UI in the block editor. If set to `false`, the user won't be able to change the lightbox settings in the block editor.
+
+
```

Comment:
I don't think the theme.json `settings.enabled` or `settings.allowEditing` options should control lightbox. This file is auto-generated from the theme.json schema, so you'll need to update things there.

Reply:
Hmm I think this might not be getting generated properly because the lightbox settings are not linked to any corresponding values in `styles`, though I'm not entirely sure.

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/54071/commits/11e96e2b2d1b27a160501cbe571e7dbb51c019b6